db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql

```diff
@@ -0,0 +1,10 @@
+\c mosip_kernel sysadmin
+
+DROP TABLE IF EXISTS kernel.key_alias CASCADE;
+DROP TABLE IF EXISTS kernel.key_policy_def CASCADE;
+DROP TABLE IF EXISTS kernel.key_policy_def_h CASCADE;
+DROP TABLE IF EXISTS kernel.key_store CASCADE;
+DROP TABLE IF EXISTS kernel.sync_control CASCADE;
+DROP TABLE IF EXISTS kernel.sync_job_def CASCADE;
+DROP TABLE IF EXISTS kernel.sync_transaction CASCADE;
+DROP TABLE IF EXISTS kernel.dao_key_store CASCADE;
```

Comment:
As per 1.1.5.5 I don't see these tables in mosip_kernel, so we have to remove this section completely